src/cmd/ksh93/tests/path.sh

```diff
@@ -265,11 +265,11 @@ status=$($SHELL -c $'trap \'print $?\' ERR;/dev/null 2> /dev/null')
 
 # universe via PATH
 
-builtin getconf
-getconf UNIVERSE - att # override sticky default 'UNIVERSE = foo'
+#getconf
+#getconf UNIVERSE - att # override sticky default 'UNIVERSE = foo'
```

Comment:
This should probably just be removed as I don't believe there is any dual universe OS still in widespread use we care about.